Makefile

```diff
@@ -268,15 +222,12 @@ SERVICE_ACCOUNTS+=--extra-service-accounts dynatrace-dynakube-oneagent-unprivile
 SERVICE_ACCOUNTS+=--extra-service-accounts dynatrace-kubernetes-monitoring
 
 # Generate bundle manifests and metadata, then validate generated files.
-.PHONY: bundle bundle-olm
-# to avoid necessity of calling make bundle with OUT=olm
-bundle:
-	make bundle-olm OLM=true OUT=olm
-bundle-olm: manifests kustomize
+.PHONY: bundle
+bundle: OLM=true
+bundle: manifests kustomize
 	operator-sdk generate kustomize manifests -q --apis-dir ./src/api/
 	cd config/deploy/$(PLATFORM) && $(KUSTOMIZE) edit set image "quay.io/dynatrace/dynatrace-operator:snapshot"=$(OLM_IMAGE)
 	$(KUSTOMIZE) build config/olm/$(PLATFORM) | operator-sdk generate bundle --overwrite --version $(VERSION) $(SERVICE_ACCOUNTS) $(BUNDLE_METADATA_OPTS)
-	make OUT=all reset-kustomization-files
 	operator-sdk bundle validate ./bundle
 	rm -rf ./config/olm/$(PLATFORM)/$(VERSION)
```

Comment:
Would be nice if the lines 232 to 258 (can't select that many lines in GitHub), could be moved to scripts, e.g. python

Reply:
This code is generated and we shouldn't touch it in case it's going to change in the future.
Also, additional building dependency (python, pip modules etc.) would be a pain.

btw, you can mark multiple lines using shift: https://twitter.com/natfriedman/status/1179097330097643521

Reply:
but I agree it looks ugly :smile:.
How about splitting single Makefile to two Makefiles: generated and handmade? The latter can call generated when need.